//@execute =!BoundsCheckException
def f(x: Range<int>) {
	x[0] = 99;
}

def main() -> int {
	def x: Array<int> = null;
	f(x);
	return 11;
}
